src/main/resources/mybatis/observations.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="observations">
+
+    <select id="getObsverationsJson" parameterType="map" resultType="String">
+       select json_build_object('observationTypes', 
+          json_build_array(json_build_object(
```

Comment:
This is an expensive way to build the json output (building a string in the database when you are not actually getting any of the data from the database). However, I think in this case we should keep the database call, just add the logic to validate the collectionId/featureId to this query.